app/javascript/lib/validation/validate-file-content.js

```diff
@@ -13,6 +13,20 @@ import { readFileBytes } from './io'
 import ChunkedLineReader from './chunked-line-reader'
 import { PARSEABLE_TYPES } from 'components/upload/upload-utils'
 
+export const REQUIRED_CONVENTION_COLUMNS = [
+  'biosample_id',
+  'disease',
+  'disease__ontology_label',
+  'donor_id',
+  'library_preparation_protocol',
+  'library_preparation_protocol__ontology_label',
+  'organ',
+  'organ__ontology_label',
+  'sex',
+  'species',
+  'species__ontology_label'
+]
```

Comment:
Blocker:
* Link to authoritative list of such columns in a code comment here
* Note in that other doc with the authoritative list that the list here must be updated when that one is